bluesky/utils.py

```diff
@@ -1486,3 +1486,54 @@ def _rearrange_into_parallel_dicts(readings):
         data[key] = payload['value']
         timestamps[key] = payload['timestamp']
     return data, timestamps
+
+
+def plot_prune_fifo(bec, n, y, x):
+    """
+    find the plot with axes x and y and replot with only the last *n* lines
+
+    Note: this is not a bluesky plan.  Call it as normal Python function.
+
+    EXAMPLE::
+
+        plot_prune_fifo(bec, 1, noisy, m1)
+
+    PARAMETERS
+    
```

Comment:
Could you please change it to be consistent with the other code and be a valid sphinx doc? The EXAMPLE block may need that update too.

Reply:
Thanks for the review.  I'll make those changes.